### DISCUSSION TITLE
Why does Fingerprint unlock not available on Linux?

### DISCUSSION THREAD

**Discussion by andrewebdev** [keepassxreboot/keepassxc#11022] [Q&A] [answered]
I've seen some closed issues related to fingerprint unlock, at least one of which directly mentioning for linux. But I've yet to find a way to use it. I use fingerprint to log into my framework. But I'd really like to use it for keepass

**Comment by droidmonkey** [comment, marked as answer]
It's going to be released with version 2.8.0

**Reply by andrewebdev** [reply]
That's good to know thanks!

**Reply by Centaro0** [reply]
Almost there!

**Comment by michelangelo4** [comment]
Is this already implemented? That is, will building the github version of keepassxc include this feature?

**Reply by droidmonkey** [reply]
It will but you need to make sure the polkit policy is properly deployed.
